docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -275,11 +400,11 @@ aggregations. See
 
 [discrete]
 [[aggs-single-dfeeds]]
-== Single bucket aggregations in {dfeeds}
+== Using single bucket aggregations in {dfeeds}
 
-{dfeeds-cap} not only supports multi-bucket aggregations, but also single bucket
-aggregations. The following shows two `filter` aggregations, each gathering the
-number of unique entries for the `error` field.
+You can also use single bucket aggregations in {dfeeds}. The example below shows 
```

Comment:
I recently read we're trying to avoid directional language (per our accessibility content design guidelines), so this caught my eye:
```suggestion
You can also use single bucket aggregations in {dfeeds}. The following example shows 
```